lib/pbench/server/api/resources/users_api.py

```diff
@@ -1,6 +1,6 @@
 from http import HTTPStatus
 import jwt
-from flask import request, jsonify, make_response, app
+from flask import request, jsonify, make_response, current_app as app
```

Comment:
@dbutenhof, @npalaska do either of you feel like `app` is too generic a name?  I would vote for `flask_app`, myself.

(That is, by the time I get to line 18, it's not obvious to me that `app` refers to Flask's object...now, that could just be my limited experience with Flask (I generally do my best to ignore the fact that it is up there somewhere in my call tree...), so I figure you guys would know better.)